src/utils/gaa-test.js

```diff
@@ -1979,13 +1979,21 @@ describes.realWin('GaaGoogle3pSignInButton', {}, () => {
       clock.tick(100);
       await tick(10);
 
+      // Send intro post message.
+      postMessage({
+        stamp: POST_MESSAGE_STAMP,
+        command: POST_MESSAGE_COMMAND_INTRODUCTION,
+      });
+
+      // Wait for promises and intervals to resolve.
+      clock.tick(100);
+      await tick(10);
+
       // Click button.
       self.document.getElementById(GOOGLE_3P_SIGN_IN_BUTTON_ID).click();
       clock.tick(100);
       await tick(10);
 
-      expect(self.open).to.have.been.calledOnce;
```

Comment:
This expectation seemed redundant, since subsequent tests specifically expect this to be called in certain ways

Reply:
Also this expectation didn't seem to match the test description